inc/class-workflow.php

```diff
@@ -157,7 +157,7 @@ public function when( $event ): Workflow {
 				if ( isset( $listener['callback'] ) && is_callable( $listener['callback'] ) ) {
 					$result = call_user_func_array(
 						$listener['callback'],
-						array_merge( $args, [ 'ui_data' => $ui_data ] )
+						array_values( array_merge( $args, [ 'ui_data' => $ui_data ] ) )
```

Comment:
It would also work to do this:

```suggestion
						array_merge( $args, [ $ui_data ] )
```

I think are a couple of other occurrences of this code to update as well

Reply:
Thanks Rob, I have updated the others with your suggestion made here.